src/agora/docoding.cpp

```diff
@@ -52,26 +55,27 @@ Event_data DoEncode::launch(size_t tag)
     int8_t* input_ptr
         = cfg->get_info_bits(raw_data_buffer_, symbol_idx_dl, ue_id, cur_cb_id);
 
+    //printf("^^^^^^^^^^^^^Sizes %d, %d, %zu", LDPC_config.Bg, LDPC_config.Zc, LDPC_config.nRows);
     ldpc_encode_helper(LDPC_config.Bg, LDPC_config.Zc, LDPC_config.nRows,
-        encoded_buffer_temp, parity_buffer, input_ptr);
+        encoded_buffer_temp, parity_buffer, input_ptr); /* overrun */
```

Comment:
What does the overrun comment mean?

Reply:
I'll remove this commit.  I marked a few places that had memory overruns in certain configurations.  I will be fixing the use cases I found in the next pr for the combined Uplink/Downlink